docs/guides/deployment/kubernetes.md

```diff
@@ -41,12 +41,47 @@ Let's start by creating a new cluster. If you already have one, skip ahead to th
     ```bash
     # Replace the cluster name with your own value
     eksctl create cluster --fargate --name <CLUSTER-NAME>
+    # Authenticate to the cluster (unless eksctl does this for you automatically)
```

Comment:
@EmilRex - are there any additional details we should add here to guide the user to authenticate with their nely created EKS cluster?

Reply:
Ah, yep actually... realizing I didn't document that well. We should probably include the standard command:
```bash
aws eks update-kubeconfig --name <CLUSTER-NAME>
```